modules/compute/container-registry/version.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
+  "version": "1.0",
```

Comment:
Can we change the version to 0.1?

Reply:
As discussed we live it as is to keep in sync with the original modules.